scripts/travis_yml_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test to check that travis.yml file & protractor.conf.js have the
+same test suites"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+from core.tests import test_utils
+import python_utils
+import utils
+
+
+def extract_travis_jobs_section():
+    """Extracts the env/jobs section from the .travis.yml file.
+
+    Returns:
+        list. A list of all jobs in travis.yml file.
+    """
+    travis_file = read_travis_yml_file()
+    jobs_raw = travis_file['env']['jobs']
+    suites_from_jobs = []
+    for job in jobs_raw:
+        suites_from_jobs.append(utils.snake_case_to_camel_case(job[14:-5].lower()))
+    return suites_from_jobs
+
+
+def extract_travis_script_section():
+    """Extracts the script section from the .travis.yml file.
+
+    Returns:
+        list. A list of all scripts in travis.yml file.
+    """
+    travis_file = read_travis_yml_file()
+    script_raw = travis_file['script']
+    # extracting test suite from patterns like --suite="accessibility"
+    hyphen_between_regex = re.compile(r'--(.+?)--')
```

Comment:
This is too general; it could plausibly match other lines besides the ones you're interested in if the Travis script section is modified in the future. See if you can write a more specific regex that has a high chance of capturing exactly the lines we are interested in, even if the script gets modified in future.

Reply:
Made it more specific.
Done.